elements/lisk-p2p/test/functional/_global_hooks.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+beforeEach(() => {
+	sandbox.restore();
```

Comment:
does functional test use `sandbox`?

Reply:
It does not, but the global mocha `_setup.ts` need this.

Reply:
_global_hook is suppose to be read automatically by `mocha` and in `_setup.ts` there is no reference to this file

Reply:
I made a new issue about this #4416.